cmd/cloud_login.go

```diff
@@ -29,16 +29,16 @@ func getCmdCloudLogin(gs *state.GlobalState) *cobra.Command {
 	// loginCloudCommand represents the 'cloud login' command
 	exampleText := getExampleText(gs, `
   # Prompt for a Grafana Cloud k6 token
-  {{.}} cloud login
+  $ {{.}} cloud login
```

Comment:
Leftover from another PR, just trying to keep all commands coherent in style. I think it's small and scoped enough to be added here instead of creating a PR specifically for that.